internal/services/exchange.service.go

```diff
@@ -8,81 +8,85 @@ import (
 	"momentum-go-server/internal/utils"
 	"net/http"
 	"os"
-	"slices"
-	"strings"
 	"time"
 
 	_ "github.com/joho/godotenv/autoload"
 )
 
 var currentTime = time.Now()
 var yesterday = currentTime.AddDate(0, 0, -1)
-var symbolsArr = []string{"AUD", "BRL", "BTC", "CAD", "CLP", "CNY", "CZK", "EGP", "EUR", "GBP", "HKD", "INR", "JPY", "KRW", "LTL", "LVL", "TRY", "USD", "XAG", "XAU"}
 
-func getNBUData(date string) []models.NBU {
+// var symbolsArr = []string{"AUD", "BRL", "EGP", "CAD", "CLP", "CNY", "CZK", "EGP", "EUR", "GBP", "HKD", "INR", "JPY", "KRW", "LTL", "LVL", "TRY", "USD", "XAG", "XAU", "UAH", "PLN"}
```

Comment:
Better write it in such way: Available currencies: []string{ "AUD", ....

Reply:
Fixed: Deleted this comment